components/service/glean/src/main/java/mozilla/components/service/glean/storages/GenericStorageEngine.kt

```diff
@@ -255,11 +255,11 @@ internal abstract class GenericScalarStorageEngine<ScalarType> : StorageEngine {
      *        the new one; this allows to implement new behaviours such as adding.
      */
     @Synchronized
-    protected fun recordScalar(
+    protected fun recordMetric(
         metricData: CommonMetricData,
-        value: ScalarType,
+        value: MetricType,
         extraSerializationData: Any? = null,
-        combine: ScalarRecordingCombiner<ScalarType>
+        combine: GleanRecordingCombiner<MetricType>
```

Comment:
nit: what about calling this  `MetricsCombiner`?

Reply:
Sounds good to me!